src/main/kotlin/com/soywiz/korlibs/modules/Bintray.kt

```diff
@@ -9,9 +9,6 @@ import java.net.URL
 import java.util.*
 
 fun Project.configureBintrayTools(ci: CI = CI(project.version)) {
-    val projectBintrayOrg by lazy {
-        findProperty("project.bintray.org")?.toString() ?: error("Can't find project.bintray.org")
```

Comment:
What prevents you to just put the same user in the `project.bintray.org` property?

Reply:
Actually it's what I did for publish, but didn't think at that ad first